recipes/copilot

```diff
@@ -0,0 +1 @@
+(mint-mode :fetcher github :repo "https://github.com/copilot-emacs/copilot.el")
```

Comment:
`s/mint-mode/copilot/` ?

Reply:
Oops, just force pushed. Thanks for catching that.

I copy pasted too quickly from an old MELPA submission of mine 🦆